ppr-ui/src/App.vue

```diff
@@ -271,9 +271,11 @@ export default class App extends Mixins(AuthMixin) {
         }
       }
 
-      // if we are already authenticated then go right to init
+      // When we are authenticated, allow time for session storage propagation from auth, then initialize application
       // (since we won't get the event from Signin component)
-      if (this.isAuthenticated) this.onProfileReady(true)
+      if (this.isAuthenticated) {
+        setTimeout(() => { this.onProfileReady(true) }, this.isJestRunning ? 0 : 1000)
```

Comment:
Is 1 sec a min we could go, have you tried a lower number? Not a big deal but maybe we could save 'some' time 😃

Reply:
hahah potentially! I didn't test every milli increment and this could vary between environments as well.
I promise you, 1s wasn't noticeable 😆

Reply:
😆 haha ok sounds good!